core/templates/dev/head/pages/about/about.html

```diff
@@ -408,6 +406,10 @@
                 Rahim Nathwani, Robyn Choo, Tricia Ngoon, Vikrant Nanda,
                 Vinamrata Singal and Yarin Feigenbaum.
               </p>
+
+              <p>
+                We would also like to thank <a href="https://translatewiki.net">translatewiki.net</a> for providing crowdsourced translations.
```

Comment:
Maybe this is a better page to link to? https://translatewiki.net/wiki/Translating:Oppia